braid/_braid.c

```diff
@@ -2924,7 +3041,7 @@ _braid_FRefine(braid_Core   core,
 
    /* Post receives */
    recv_size = f_npoints*(1+isize+1); /* max receive size */
-   recv_buffer = _braid_CTAlloc(braid_Real, recv_size+1);
+   recv_buffer = _braid_CTAlloc(braid_Real, recv_size);
    f_ca = _braid_CTAlloc(braid_Int,  f_npoints);
```

Comment:
The MPI_Recv below puts (recv_size+1) data elements into recv_buffer.  Don't we want this +1 ?

Reply:
Actually, the MPI_Recv only needs size, not (1+size), because size already accounts for the extra message-length data I prepend to each send buffer.  I suppose that could cause an implementation of MPI to do something illegal with the buffer memory and cause a memory problem, even though there should be no actual messages received that would overrun the recv_buffer.  I'll try making that change and see if I am correct here.

Reply:
Changed (1+size) to size in the MPI_Recv.  It works for me, valgrind turned on.